### ISSUE TITLE
Models providing seemingly random text?

### ISSUE THREAD

**Issue by mhaddon** [go-skynet/LocalAI#180]
I couldnt run gpt4all-j model for the same reason as the people in this thread: https://github.com/go-skynet/LocalAI/issues/88

However, I can run other models, like ggml-gpt4all-l13b-snoozy.bin, ggml-vicuna-7b-1.1-q4_2.bin etc.

But when I do the api responds the weirdest text.

```
curl http://localhost:8080/v1/chat/completions -H "Content-Type: application/json" -d '{
     "model": "ggml-gpt4all-l13b-snoozy.bin",
     "messages": [{"role": "user", "content": "What is Kubernetes?"}],
     "temperature": 0.7
   }'

{"object":"chat.completion","model":"ggml-gpt4all-l13b-snoozy.bin","choices":[{"message":{"role":"assistant","content":"Yes, there are several other ways to generate random numbers in Python. Here are a few examples:\n```python\nimport random\n\n# Generate a list of random integers between 1 and 6\nrandom.sample(range(1, 7), 3)\n\n# Generate a string of random characters\nimport string\nimport random\n\n# Generate a string of length 5 containing only letters (a-z)\nrandom_string = ''.join([random.choice(string.ascii_uppercase) for _ in range(5)])\n```"}}],"usage":{"prompt_tokens":0,"completion_tokens":0,"total_tokens":0}}% 
```

```
curl http://localhost:8080/v1/completions -H "Content-Type: application/json" -d '{
     "model": "ggml-gpt4all-l13b-snoozy.bin",
     "prompt": "What is a cat?",
     "temperature": 0.7
   }'
{"object":"text_completion","model":"ggml-gpt4all-l13b-snoozy.bin","choices":[{"text":"The question is about the origin of life on Earth, specifically the role of the Big Bang and abiogenesis in shaping the conditions for life to emerge."}],"usage":{"prompt_tokens":0,"completion_tokens":0,"total_tokens":0}}%
```

**Comment by mudler** [comment]
Hi 👋 

Did you provided the gpt4all prompt template for the model?

**Comment by mudler** [comment]
I think there was a regression as the input was cleared up if the model template is missing. Fix is in #187

**Comment by mudler** [comment]
That should be fixed now. cutting a new patch release - please feel free to reopen if it doesn't work!
